File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> map = new HashMap<>();
        for(Integer i = 0 ; i < nums.length ; i++){
            Integer element = target - nums[i];
            if (map.containsKey(element)){
                return new int[] { map.get(element), i };
            }
            else{
                map.put(nums[i],i);
            }
        }
        return new int[] { };


            }
}

File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                total=nums[i]+nums[j]
                if(total == target):
                    return i,j
                    break






        
        